### PR TITLE
add Authorization to the default headers for CrossOriginFilter

### PR DESCRIPTION
**Pull request by jmcampanini**
@derekcicerone + @bavardage for SA

### CODE REVIEW

**Comment by bavardage** [comment]
sweet
:+1:

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The following are the default values, only specify values if they differ from th
 webSecurity:
   cors:
     allowCredentials: true
-    allowedHeaders: "X-Requested-With,Content-Type,Accept,Origin"
+    allowedHeaders: "X-Requested-With,Content-Type,Accept,Origin,Authorization"
```
Comment (derekcicerone-zz) [inline README.md]: can we sort these? :)

Comment (jmcampanini) [inline README.md]: sure! :) it came with that funky ordering from the `CrossOriginFilter` values. sorting it seems sane though - easier to spot a specific header.

**Comment by derekcicerone-zz** [comment]
Looks good, thanks javier!

**Pull request merged**